react/src/styles/App.module.scss

```
@@ -1526,6 +1526,35 @@ a {
             line-height: $line-height-normal + 4px;
         }
     }
+
+    &--privacy {
+        line-height: $line-height-big;
```

```suggestion
        line-height: $line-height-normal + 4px;
```
$line-height-normal + 4px 로 부탁드려요. 리뷰나 공지 등 긴 본문에 적용하는 line-height입니다